services/provider/server/server.go

```diff
@@ -814,24 +816,36 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 					}),
 					Labels: getExternalResourceLabels("VolumeGroupSnapshotClass", mirroringEnabled, false, replicationID, storageID)},
 				&pb.ExternalResource{
-					Name: "ceph-rbd",
+					Name: req.StorageClaimName,
```

Comment:
Like all other classes, our VRCs should have a fixed name and not one that depends on the request